examples/webbilling-demo/src/tests/tax-calculation.test.ts

```diff
@@ -42,433 +49,541 @@ const navigateToTaxesLandingUrl = (page: Page, userId: string) =>
+            calculateTaxesCount++;
+            await route.fallback();
+          });
+
+          await enterEmail(page, email);
+          await enterCreditCardDetails(
+            page,
+            "4242 4242 4242 4242",
+            ITALY_CUSTOMER_DETAILS,
+          );
 
-      await expect(page.getByText("Total excluding tax")).toBeVisible();
+          await confirmTaxCalculation(page);
+          await expect(page.getByText(/VAT - Italy/)).toBeVisible();
+          await expect(calculateTaxesCount).toBe(1);
 
-      let calculateTaxesCount = 0;
-      await page.route(TAX_ROUTE_PATH, async (route) => {
-        calculateTaxesCount++;
-        await route.continue();
-      });
+          await enterCreditCardDetails(
+            page,
+            "4242 4242 4242 4242",
+            SPAIN_CUSTOMER_DETAILS,
+          );
 
-      await enterEmail(page, email);
-      await enterCreditCardDetails(
-        page,
-        "4242 4242 4242 4242",
-        ITALY_CUSTOMER_DETAILS,
-      );
+          await confirmTaxNotCalculating(page);
+          await expect(page.getByText(/VAT - Spain/)).not.toBeVisible();
+          await expect(calculateTaxesCount).toBe(1);
 
-      await confirmTaxCalculation(page);
-      await expect(page.getByText(/VAT - Italy/)).toBeVisible();
-      await expect(calculateTaxesCount).toBe(1);
+          await clickPayButton(page);
 
-      await enterCreditCardDetails(
-        page,
-        "4242 4242 4242 4242",
-        SPAIN_CUSTOMER_DETAILS,
+          await expect(
+            page.getByText(
+              /The total price was updated with tax based on your billing address/i,
+            ),
+          ).not.toBeVisible();
+          await expect(page.getByText(/VAT - Spain/)).toBeVisible();
+          await expect(calculateTaxesCount).toBe(2);
+        },
       );
-
-      await confirmTaxNotCalculating(page);
-      await expect(page.getByText(/VAT - Spain/)).not.toBeVisible();
-      await expect(calculateTaxesCount).toBe(1);
-
-      await clickPayButton(page);
-
-      await expect(
-        page.getByText(
-          /The total price was updated with tax based on your billing address/i,
-        ),
-      ).not.toBeVisible();
-      await expect(page.getByText(/VAT - Spain/)).toBeVisible();
-      await expect(calculateTaxesCount).toBe(2);
     },
   );
+});
 
-  integrationTest("Stripe tax not active", async ({ page, userId }) => {
+integrationTest.describe("Tax calculation setup errors", () => {
+  integrationTest.fixme("Stripe tax not active", async ({ page, userId }) => {
     await page.route(TAX_ROUTE_PATH, async (route) => {
       await route.fulfill({
-        body: '{ "code": 7898, "message": "Stripe account setup error: Stripe Tax must be active to calculate taxes."}',
+        json: {
+          mocked: true,
+          code: 7898,
+          message:
+            "Stripe account setup error: Stripe Tax must be active to calculate taxes.",
+        },
         status: 422,
```

Comment:
We already have fixtures for these fulfillments, right? `STRIPE_TAX_NOT_ACTIVE_RESPONSE`